presto-docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -215,16 +215,110 @@ The connector maps PrestoDB types to the corresponding MySQL types:
 
 No other types are supported.
 
+SQL Support
+-----------
+
+The MySQL connector allows querying and creating MySQL tables. Here are some examples of the SQL operations supported:
+
+ALTER TABLE
+^^^^^^^^^^^
+
+.. code-block:: sql
+
+    ALTER TABLE mysql.web.page_views ADD COLUMN zipcode VARCHAR;
+    ALTER TABLE mysql.web.page_views RENAME COLUMN zipcode TO location;
+    ALTER TABLE mysql.web.page_views DROP COLUMN location;
+
+CREATE TABLE
+^^^^^^^^^^^^
+
+Create a new MySQL table named ``page_views`` in the ``web`` schema:
+
+.. code-block:: sql
+
+    CREATE TABLE mysql.web.page_views (
+      user_id bigint,
+      page_url varchar,
+      ds date,
+      country varchar
+    );
+
+.. note:: Presto does not enforce primary key constraints.
+
+.. code-block:: sql
+
+   CREATE TABLE users (
+           id INT PRIMARY KEY,
+           name TEXT,
+           email TEXT
+       );
+
+.. admonition:: Error
+   :class: error
+
+   Query 20240322_095447_00010_syzb3 failed: line 2:19: mismatched input 'PRIMARY'. Expecting: ')', ','
+
+CREATE TABLE AS SELECT
+^^^^^^^^^^^^^^^^^^^^^^
+
+Create a new table ``page_views_new`` from an existing table ``page_views``:
+
+.. code-block:: sql
+
+    CREATE TABLE mysql.web.page_views_new AS SELECT * FROM mysql.web.page_views;
+
+.. note:: Advanced SQL features such as data compression are not supported in the ``CREATE TABLE AS SELECT`` statement.
```

Comment:
I suggest that you apply the same changes in the earlier comment here in lines 270-279, so this Note will include the sentence "Advanced SQL features such as data compression...", the error code block, and the error statement, like the example in the previous comment and screenshot.